pkg/util/cloudproviders/gce/gce_tags_test.go

```diff
@@ -146,3 +147,20 @@ func TestGetHostTagsWithNonDefaultTagFilters(t *testing.T) {
 	require.NoError(t, err)
 	testTags(t, tags, expectedExcludedTags)
 }
+
+func TestGetHostTagsWithProviderKind(t *testing.T) {
+	ctx := context.Background()
+	mockConfig := configmock.New(t)
+	defaultProviderKind := mockConfig.GetStringSlice("provider_kind")
```

Comment:
Is this supposed to be `GetStringSlice`? It's simply `GetString` in the non-test code.

Reply:
Thanks good catch, just updated!